mackup/applications/appcode-31.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = AppCode 3.1
+
+[configuration_files]
+Library/Application Support/appCode31
+Library/Preferences/appCode31
```

Comment:
@lra It seems github online view mode hide the last newline character automaticlly. I can see the newline character in edit mode.